amundsen_application/static/js/ducks/tableMetadata/api/v0.ts

```diff
@@ -9,50 +9,48 @@ import { PreviewData, PreviewQueryParams, TableMetadata, User, Tag } from 'inter
 
 const API_PATH = '/api/metadata/v0';
 
-type MessageResponse = { msg: string };
+type MessageAPI = { msg: string };
```

Comment:
Seems like a common interface used in most APIs. Maybe this should be shared in interfaces?

Reply:
Ah I see. But it's only used in `ducks`, and right now the current concept behind `interfaces` is that it would contain shared interfaces between `ducks` and `components`. That being said, if we identify any other things shared in `ducks`, I'm alright to start separating stuff out into a place for shared ducks types. I'll put a comment here.